commander.js

```diff
@@ -0,0 +1,23 @@
+const program = require('commander');
+program.version('0.0.1');
+
+program
+    .option('-no-commit', 'no commit step')
+    .option('--no-install', 'no install step')
+
+program.parse(process.argv);
+if (program.no-commit) {
```

Comment:
`program.no-commit` can't possibly be the variable, that would evaluate as `program.no - commit`.

Reply:
according to the docs, this should be checking `program.commit`